docs/design/TEMPLATE.md

```diff
@@ -1,12 +1,8 @@
-<!--
-This is a template for TiDB's change proposal process, documented [here](./README.md).
--->
+# TiDB Design Documents
 
-# Proposal: <!-- Title -->
-
-- Author(s):     <!-- Author Name, Co-Author Name, with the link(s) of the GitHub profile page -->
-- Last updated:  <!-- Date -->
-- Discussion at: <!-- https://github.com/pingcap/tidb/issues/XXX -->
+- Author(s): @ghid1, @ghid2, ...
+- Proposal PR: https://github.com/pingcap/tidb/pull/XXX
```

Comment:
This field is only determined when the PR is filed. It's better to point to the issue which describes the problem this proposal aimed to solve.

Reply:
The tracking issue is mentioned just the next line. And the pr number can be updated once it is created.